webboot/webboot.go

```diff
@@ -212,9 +212,13 @@ func main() {
 			log.Fatal(err)
 		}
 	}
-
+	//the Request function sets up a DHCP confifuration for all interfaces,
+	//such as eth0, which is an ethernet interface.
 	if *ipv4 || *ipv6 {
+		//ifname uses the regular expression ^[we].* to check for wlan0 or eth0 interfaces.
```

Comment:
that re matches more than just wlan0 and eth0 right?

Reply:
well i don’t know what exactly w and e can stand for, but i know wlan and eth0 are options. maybe those are just examples but i don’t know what else could be.